### PR TITLE
Deprecate Bio.GA for biopython 1.70 

### PR DESCRIPTION
**Pull request by Adil-Iqbal**
- Deprecation warning added to GA module.
- Deprecation warning caught by all GA unit tests.
- Deprecation info added to Deprecation.rst

### CODE REVIEW

**Comment by Adil-Iqbal** [comment]
I don't quite know how to resolve this issue. I forked the most recent version of biopython before I made my edits. Is that a problem?

File: Bio/GA/__init__.py [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+"""Genetic Algorithm library (DEPRECATED)."""
+
+import warnings
+from Bio import BiopythonDeprecationWarning
+warnings.warn("Bio.GA has been deprecated, and we intend to remove"
+              " it in a future release of Biopython. Please consider using"
+              " DEAP instead.  If you would like to"
+              " continue using Bio.SomeModule, please contact the Biopython"
```
Comment (peterjc) [inline Bio/GA/__init__.py]: That should be Bio.GA, not Bio.SomeModule

Comment (Adil-Iqbal) [inline Bio/GA/__init__.py]: Changes made in most recent commit.

**Comment by Adil-Iqbal** [comment]
Yes, I am happy for all of that. The dual license AND the thank you mention are much appreciated.

I hereby give my explicit permission to switch to the BSD 3-clause license.

**Comment by Adil-Iqbal** [comment]
Changes made. Still don't know why Codev is acting up.

**Comment by peterjc** [comment]
Don't worry about CodeCov for this pull request.

CodeCov is unhappy because at the time you created your branch, the master branch was failing - so for this pull request we have no coverage information to compare against. The master branch is now fixed https://github.com/biopython/biopython/commit/ac7afe872efdef96f12af8b3256687171b982856 other than a separate problem with tests timing out...

**Comment by peterjc** [comment]
Thank you very much :)

**Pull request merged**